python/raft-ann-bench/src/raft-ann-bench/run/__main__.py

```diff
@@ -173,13 +184,17 @@ def main():
         type=positive_int,
         help="number of query vectors to use in each query trial",
     )
+    parser.add_argument(
+        "--dataset-configuration",
+        help="path to configuration file for datasets",
+    )
     parser.add_argument(
         "--configuration",
-        help="path to configuration file for a dataset",
+        help="path to configuration file or directory for algorithms",
```

Comment:
Can you just briefly point out that any groups found in the specified files/directories wil automatically override groups of the same name for corresponding algorithms? (Including 'base')